styles.css

```diff
@@ -1,10 +1,8 @@
-body {
-    color: white;
-}
 
 .item {
     border: 1px solid var(--background-modifier-border);
     padding: 10px;
+    color: var(--text-normal);

```

Comment:
What's the rationale behind using var(--text-normal) instead of setting the color manually? I'm assuming that it will use the color from the currently selected theme. Is this correct?

Reply:
Yeah, that's exactly what I was going for